src/idl_gen_pybind.cpp

```diff
@@ -0,0 +1,1259 @@
+/*
+ * Copyright 2024 Figure AI, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+TODO(michaelahn): Feature completion:
+- Include docstrings in py::doc.
+- (In)equality operators.
+- Vector __contains__, more sequence methods.
+- C++-specific flatbuffer features (native types).
+*/
+
+#include "idl_gen_pybind.h"
+
+#include <set>
+#include <unordered_map>
+#include <unordered_set>
+#include <vector>
+
+#include "flatbuffers/code_generators.h"
+#include "flatbuffers/flatbuffers.h"
+#include "flatbuffers/flatc.h"
+#include "flatbuffers/idl.h"
+#include "flatbuffers/util.h"
+#include "idl_namer.h"
+
+namespace flatbuffers {
+namespace pybind {
+
+namespace {
+
+// Taken from idl_gen_cpp.cpp.
+const std::unordered_set<std::string> &CppKeywords() {
+  const auto *const kKeywords = new std::unordered_set<std::string>{
+    "alignas",
```

Comment:
Maybe we can create a keywords.h with both of these in it and use them here and in the _cpp.cpp and _python.cpp

Reply:
+1, I'd like this but perhaps I'll refactor this closer to upstreaming as I'd like to avoid touching existing flatbuffer generator code if possible in this PR.